learn/quick-start.md

```diff
@@ -1,7 +1,142 @@
 # Quick start
 
- Work in progress :construction:
+ To start to use Codex we would need to perform the following steps
+ - [Get Codex binary](#get-codex-binary)
+ - [Run Codex](#run-codex)
+ - [Interact with Codex](#interact-with-codex)
 
- Please read
- - [Codex Two-Client Test](https://github.com/codex-storage/nim-codex/blob/master/docs/TwoClientTest.md)
- - [Running a Local Codex Network with Marketplace Support](https://github.com/codex-storage/nim-codex/blob/master/docs/Marketplace.md)
+### Get Codex binary
+
+ For quick start we would use precompiled binaries from [GitHub release page](https://github.com/codex-storage/nim-codex/releases). If you rather prefer to compile from the sources, please check [Build Codex](/learn/build).
+
+ 1. Download binary and checksum for your platform/architecture
+
+    **Linux/macOS**
+    ```shell
+    version=v0.1.3
+    platform=linux     # darwin
+    architecture=amd64 # arm64
+
+    # Binary
+    curl -LO https://github.com/codex-storage/nim-codex/releases/download/${version}/codex-${version}-${platform}-${architecture}.tar.gz
+
+    # Checksum
+    curl -LO https://github.com/codex-storage/nim-codex/releases/download/${version}/codex-${version}-${platform}-${architecture}.tar.gz.sha256
+    ```
+
+ 2. Verify checksum
```

Comment:
Do we know how to do this on win?

Reply:
Windows is not covered in this guide at all, but all should work on Linux/macOS.

We would need to add all the commands for that platform and check which tools do we have there.

We can do this in this PR or in a next one.